source/install.txt

```diff
@@ -45,6 +45,16 @@ Select the appropriate tab for your operating system:
    .. tab::
       :tabid: macos
 
+      Platform Support
+      ~~~~~~~~~~~~~~~~
+      
+      ``mongosh`` supports Apple silicon starting with ``mongosh``
+      0.13.2.
+
+      :manual:`Automatic client-side field level encryption
+      </core/security-automatic-client-side-encryption/>` is not
+      supported on Apple silicon.
```

Comment:
This ticket was created over a year ago – while this paragraph *technically* still accurate, it will also stop being true within the next week or two, so I’d just drop it :slightly_smiling_face: